projects/portfolio/portfolio-webpage-1.md

```diff
@@ -124,16 +124,16 @@ You should write clear + informative commit messages for each commit and create
 
 **README**
 
-- [ ] Create a README.md file with your command line.
+- [ ] Create a README.md file from your command line.
 - [ ] Include relevant README content (what the project is, what open-source license it uses, how to install, etc.).
-- [ ] Use markdown to organize your README.
+- [ ] Use markdown language code to organize your README.
 
 ---
 
 ### Recommended extension activity if time permits
 
-- [ ] Move your portfolio project into its own repo instead of within the techtonica-assignments repo
 - [ ] Go into the settings of your portfolio-website repo in GitHub and create a link for your live site
-- [ ] Link to your live github.io site clickable on your README.
+- [ ] Make sure the link to your live github.io site is clickable on your README.
```

Comment:
```suggestion
- [ ] Link to your live github.io site is clickable on your README.
```
this can remain in the same formal, instructional tone

Reply:
@daaimah123 Done! I also added 'that' to make the sentence grammatically correct.

"Link to your live github.io site that is clickable on your README."